docs/api-documentation/rpc.md

```diff
@@ -50,64 +68,177 @@ TBD
-* `call/<account_id>/<method name>` - calls `<method name>` in contract `<account_id>` as view function with `data` as parameters.
+### Account Information (`account/<account_id>`)
 
-`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=query params:="[\"account/test.near\",\"\"]" id="dontcare"`
+`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=query id=dontcare 'params:=["account/test.near", ""]'`
 
 ```javascript
 {
     "id": "dontcare",
     "jsonrpc": "2.0",
     "result": {
-        "code": 0,
-        "codespace": "",
-        "height": 0,
-        "index": -1,
-        "info": "",
-        "key": "YWNjb3VudC90ZXN0Lm5lYXI=",
-        "log": "exists",
-        "proof": [],
-        "value": "eyJhY2NvdW50X2lkIjoidGVzdC5uZWFyIiwibm9uY2UiOjAsImFtb3VudCI6MTAwMDAwMDAwMDAwMCwic3Rha2UiOjUwMDAwMDAwLCJwdWJsaWNfa2V5cyI6W1sxNjIsMTIyLDE0MCwyMTksMTcyLDEwNSw4MCw3OCwxOTAsMTY1LDI1NSwxNDAsMTExLDQzLDIyLDE0OSwyMTEsMTUyLDIyNywyMjcsNjcsMjIyLDIzNCw3Nyw5NiwxNTYsNjYsMjMsMTcyLDk2LDc2LDEzN11dLCJjb2RlX2hhc2giOiJBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBPSJ9"
+        "amount": "1000000000000000011",
+        "code_hash": "11111111111111111111111111111111",
+        "staked": "0",
+        "storage_paid_at": 0,
+        "storage_usage": 182
     }
 }
 ```
 
-Where `value` is base58 encoded JSON of the account status: `'{"account_id":"test.near","nonce":0,"amount":1000000000000,"stake":50000000,"public_keys":[[...]],"code_hash":"AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="}'`
+### Account Access Keys (`access_key/<account_id>`)
 
-the `[...]` will  be replaced with a public key as an array.
+`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=query id=dontcare 'params:=["access_key/drinks", ""]'`
+
+```javascript
+{
+    "id": "dontcare",
+    "jsonrpc": "2.0",
+    "result": [
+        {
+            "access_key": {
+                "nonce": 5,
+                "permission": "FullAccess"
+            },
+            "public_key": "ed25519:..."
+        },
+        {
+            "access_key": {
+                "nonce": 5,
+                "permission": "FullAccess"
+            },
+            "public_key": "ed25519:..."
+        }
+    ]
+}
+```
+
+### Account Access Key by Public Key (`access_key/<account_id>/<public_key>`)
+
+NOTE: If there is no such access key, returns nothing.
+
+`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=query id=dontcare 'params:=["access_key/drinks/ed25519:...", ""]'`
+
+```javascript
+{
+    "id": "dontcare",
+    "jsonrpc": "2.0",
+    "result": {
+        "nonce": 5,
+        "permission": "FullAccess"
+    }
+}
+```
+
+### Full State of the Contract (`contract/<account_id>`)
+
+NOTE: It might be expensive if contract has large state.
+
+`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=query id=dontcare 'params:=["contract/drinks", ""]'`
+
+```javascript
+{
+    "id": "dontcare",
+    "jsonrpc": "2.0",
+    "result": {
+        "values": {}
```

Comment:
1. It is expected that querying `crypto-corgis` account id returns an error "key must be a string"? (i.e. dash is not allowed in the API call)
2. I have failed to find a contract with non-empty `values`.